core/domain/param_domain.py

```diff
@@ -57,25 +86,58 @@ class ParamChange(object):
     """Value object for a parameter change."""
 
     def __init__(self, name, generator_id, customization_args):
-
+        """Initialize a ParamChange object with the specified arguments.
+
+        Args:
+            name: unicode. The name of the parameter.
+                generator_id: unicode. The type of generator used to create the
+                parameter, e.g.,"Copier".
+            customization_args: dict. A dict containing the following keys:
+                (value, parse_with_jinja). `value` specifies the value of the
+                parameter, and `parse_with_jinja` indicates whether parsing is
+                to be done with the Jinja template engine. If the parameter is
+                changed to one amongst several values, this dict contains a
```

Comment:
changed to one amongst several --> specified using one of several possible values